src/target/riscv/riscv.c

```diff
@@ -598,90 +606,174 @@ static int maybe_add_trigger_t1(struct target *target, struct trigger *trigger)
 	return ERROR_OK;
 }
 
-static int maybe_add_trigger_t2(struct target *target, struct trigger *trigger)
-{
-	int ret;
-	riscv_reg_t tdata1, tdata2;
+struct trigger_request_info {
+	riscv_reg_t tdata1;
+	riscv_reg_t tdata2;
+	riscv_reg_t tdata1_ignore_mask;
+};
 
+static int try_trigger_request_match(struct target *target,
+									 struct trigger *trigger, int trigger_type,
+									 struct trigger_request_info trig_info)
```

Comment:
Excessive indent here and elsewhere.

Reply:
The rest of the OpenOCD code typically uses 2 tabs when breaking statements or function declarations to more lines.

Example: https://github.com/riscv/riscv-openocd/blob/riscv/src/target/target.c#L637

Reply:
Addressed.